api/management/swagger-api_management.yaml

```diff
@@ -115,6 +115,29 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Client'
+  /realms/{realm}/trustIdGroups:
```

Comment:
In the JIRA the path is  /realms/{realm}/users/trustIdGroups
If it was re-discussed to change it to this path, ignore my comment

Reply:
We had to change the path in order to avoid conflict with URL of GET /realm/{realm}/users/{userID} (trustIdGroups would not be a valid userID but we preffer to avoid this conflict)
I just forgot to update the JIRA description

Reply:
OK